platform/wasm/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04
+
+RUN apt update -y
+RUN apt install -y \
+    cmake \
+    wget \
+    build-essential \
+    git \
+    llvm \
+    vim \
+    curl zip unzip tar \
+    pkg-config \
+    autoconf libtool \
+    flex bison \
+    python3 \
+    python3-pip \
+    nodejs \
+    ninja-build
+
+RUN pip3 install scons
+
+RUN git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
Add in ` --depth 1` to reduce the amount downloaded

Reply:
Done